packages/client/tests/functional/_utils/relationMode/computeReferentialActionLine.ts

```diff
@@ -3,7 +3,14 @@ type ComputeReferentialActionLine = {
   onDelete: string
 }
 
-export function computeReferentialActionLine({ onUpdate, onDelete }: ComputeReferentialActionLine): string {
+export type ReferentialActionLineOutput = { supportsRequired: boolean; referentialActionLine: string }
+
+export function computeReferentialActionLine({
+  onUpdate,
+  onDelete,
+}: ComputeReferentialActionLine): ReferentialActionLineOutput {
+  const supportsRequired = ![onUpdate, onDelete].includes('SetNull')
```

Comment:
This could use a comment, it's not super obvious what it does and why

Reply:
I understand it because we're both deep into what we need to do here, but let's make it clearer

Reply:
Should be fixed now.